cmd/object-handlers.go

```diff
@@ -2807,6 +2807,11 @@ func (api objectAPIHandlers) GetObjectRetentionHandler(w http.ResponseWriter, r
 
 	retention := objectlock.GetObjectRetentionMeta(objInfo.UserDefined)
 
+	// If xml namespace is empty, set a default value before returning.
+	if retention.XMLNS == "" {
+		retention.XMLNS = "http://s3.amazonaws.com/doc/2006-03-01/"
```

Comment:
This is an incorrect fix @sinhaashish we should add this to the RetentionConfiguration struct 

Also have you verified this with AWS S3 that the namespace is correct?